lm75bd/lm75bd.c

```diff
@@ -29,6 +32,37 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
   /* Implement this driver function */
   
+  // Store value of pointer register that points to temp. register
+  uint8_t writeBuffer = 0x00;
+
+  // Set pointer register to point to temp. register
+  i2cSendTo(devAddr, &writeBuffer, 1); 
+
+  // Store temp. value retrieved from sensor
+  uint8_t readBuffer[2];
```

Comment:
in general its best to initialize your arrays to 0 like `uint8_t readBuffer[2] = {0};`